tests/src/clarity-tests.ts

```diff
@@ -126,6 +130,18 @@ describe('Clarity Types', () => {
       });
       expect(Object.keys(serializedDeserialized.data)).toEqual(lexicographic);
     });
+
+    test('StringAsciiCV', () => {
+      const str = stringAsciiCV('hello world');
+      const serializedDeserialized = serializeDeserialize(str) as StringAsciiCV;
+      expect(serializedDeserialized).toEqual(str);
+    });
+
+    test('StringUtf8CV', () => {
+      const str = stringUtf8CV('hello world');
+      const serializedDeserialized = serializeDeserialize(str) as StringUtf8CV;
+      expect(serializedDeserialized).toEqual(str);
+    });
```

Comment:
Should we have a test with a string that includes something like an actual emoji, so we can check that this feature is working end to end?

Reply:
Will add this now!